pkg/util/versionCheck/versionCheck.go

```diff
@@ -12,9 +12,16 @@ const (
 
 	// NatsBinaryPath after v2 release.
 	NatsBinaryPath = "/nats-server"
+
+	// NightlyNatsBinaryPath is there the server is for nightly images.
+	NightlyNatsBinaryPath = "/bin/nats-server"
```

Comment:
maybe we should fix the nightly images which are made here I think, add an alias instead of adding to the operatoe: https://github.com/nats-io/jetstream/blob/master/Dockerfile

Reply:
ok let's add this back instead

Reply:
Readded.